string container_id
string name
float32 cpu_usage
float32 memory_usage
float32 memory_limit
float32 memory_percentage
string net_io
string block_io
int32 pids
